qa/vector/src/main/java/org/elasticsearch/test/knn/KnnIndexTester.java

```diff
@@ -230,9 +230,8 @@ public static void main(String[] args) throws Exception {
                 }
                 if (cmdLineArgs.forceMerge()) {
                     knnIndexer.forceMerge(indexResults);
-                } else {
-                    knnIndexer.numSegments(indexResults);
                 }
+                knnIndexer.numSegments(indexResults);                
             }
```

Comment:
maybe remove this completely from the reindex and force merge branch. That way when searching only we get to see the number of segments for the index. (so move this down one line)

Reply:
Good idea. Done 445a60a